src/tokens/lst_tokens/token_lst_add_after.c

```diff
@@ -3,15 +3,25 @@
 /*                                                        :::      ::::::::   */
 /*   token_lst_add_after.c                              :+:      :+:    :+:   */
 /*                                                    +:+ +:+         +:+     */
-/*   By: jodufour <jodufour@student.42.fr>          +#+  +:+       +#+        */
+/*   By: mcourtoi <mcourtoi@student.42.fr>          +#+  +:+       +#+        */
 /*                                                +#+#+#+#+#+   +#+           */
 /*   Created: 2023/03/17 18:06:53 by mcourtoi          #+#    #+#             */
-/*   Updated: 2023/03/31 07:02:23 by jodufour         ###   ########.fr       */
+/*   Updated: 2023/04/03 17:56:39 by mcourtoi         ###   ########.fr       */
 /*                                                                            */
 /* ************************************************************************** */
 
 #include "minishell.h"
 
+/**
+ * @brief Add a node t_token containing given type
+ * 		and str after the t_token prev given in parameters.
+ * 
+ * @param list token_lst to which we add the new node.
+ * @param prev token to add after.
```

Comment:
This is not relevant to what `prev` is truly meant to be. Reading this, we may understand that `prev` actually is the node to add to the list.